app/imports/ui/components/Dashboard.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {Grid, Header} from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+
+/** Renders a single row in the List Stuff table. See pages/ListStuff.jsx. */
+class Dashboard extends React.Component {
+
+  render() {
+    return (
+        <Grid.Row columns={3}>
+          <Grid.Column>
+            <Header as='h2' inverted>{this.props.dashboard.activeContracts}</Header>
```

Comment:
This is good. If you wanted to , you could also look at the Statistic component in semantic ui, which seems to fit perfectly for this use case . https://react.semantic-ui.com/views/statistic/